webrender/src/glyph_rasterizer.rs

```diff
@@ -296,7 +296,7 @@ impl FontContext {
     fn add_font(&mut self, font_key: &FontKey, template: &FontTemplate) {
         match template {
             &FontTemplate::Raw(ref bytes, index) => {
-                self.add_raw_font(&font_key, &**bytes, index);
+                self.add_raw_font(&font_key, bytes.clone(), index);
```

Comment:
nit: `Arc::clone(&bytes)` ftw (it's even [documented as the idiomatic way](https://doc.rust-lang.org/std/sync/struct.Arc.html#cloning-references) now.